BMM.Core/Implementations/Player/ViewModelHandlingMediaPlayerDecorator.cs

```diff
@@ -87,13 +91,15 @@ public async Task Play(IList<IMediaTrack> mediaFiles, IMediaTrack currentTrack,
         {
             if (!ShowErrorIfOutdatedLiveRadio(currentTrack))
             {
+                _playbackHistoryService.AddPlayedTrack(currentTrack);
```

Comment:
Sorry, I probably didn't explain this properly.
The PBI says that we should store "latest position". My idea was that the user can continue the track where he left off. That's very relevant for speeches that are often around 30 minutes long.
However to store the latest position we would need to add the track to the history AFTER he is done playing.

We also need something in the ViewModel to continue playing where the user left off.